packages/flutter/test/cupertino/text_selection_toolbar_test.dart

```diff
@@ -497,4 +502,50 @@ void main() {
     shadow = boxDecoration.boxShadow!.first;
     expect(shadow.offset.dy, equals(0.0));
   }, skip: kIsWeb); // [intended] We do not use Flutter-rendered context menu on the Web.
+
+  testWidgetsWithLeakTracking('Basic golden tests', (WidgetTester tester) async {
+    final Key key = UniqueKey();
+    Widget buildToolbar(Brightness brightness, Offset offset) {
+      final Widget toolbar = CupertinoTextSelectionToolbar(
+        anchorAbove: offset,
+        anchorBelow: offset,
+        children: <Widget>[
+          CupertinoTextSelectionToolbarButton.text(onPressed: () {}, text: 'Lorem ipsum'),
+          CupertinoTextSelectionToolbarButton.text(onPressed: () {}, text: 'dolor sit amet'),
+          CupertinoTextSelectionToolbarButton.text(onPressed: () {}, text: 'Lorem ipsum \ndolor sit amet')
+        ],
```

Comment:
Throw in a `CupertinoTextSelectionToolbarButton.buttonItem` maybe as it's the other button type used on iOS?